pkg/scheduler/plugins/gang/gang.go

```diff
@@ -19,6 +19,8 @@ package gang
 import (
 	"fmt"
 
+	"volcano.sh/volcano/pkg/scheduler/plugins/util/gangextender"
```

Comment:
It is suggested that the current imported information can be adjusted down a few lines and classified together

Reply:
Done